src/client/automation/playback/click/select-child.js

```diff
@@ -167,7 +160,7 @@ export default class SelectChildClickAutomation {
 
             // TODO: remove detection of Chrome version after fix
             // https://github.com/DevExpress/testcafe-hammerhead/issues/777
-            if ((browserUtils.isSafari || browserUtils.isWebKit && getChromeVersion() >= 53)
+            if ((browserUtils.isSafari || browserUtils.isWebKit && browserUtils.version >= 53)
```

Comment:
To get rid of this function, cause it implemented in bowser package
https://github.com/DevExpress/testcafe/pull/1070/files#diff-dec5c80a08927d045521c13f0e9ba927L25